apps/dotlottie-playground/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "run-dev": "vite --mode=development",
-    "build": "tsc && vite build",
+    "run-build": "tsc && vite build",
```

Comment:
I think we should keep build for playground for the playground deploy

Reply:
I've temporarily disabled it because the player usage in the playground needed to be updated. It's up to date now, so please take a look.